test/vstest.ProgrammerTests/Fakes/FakeProcessHelper.cs

```diff
@@ -0,0 +1,124 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace vstest.ProgrammerTests.Fakes;
+
+using Microsoft.VisualStudio.TestPlatform.PlatformAbstractions;
+using Microsoft.VisualStudio.TestPlatform.PlatformAbstractions.Interfaces;
+
+internal class FakeProcessHelper : IProcessHelper
+{
+    // starting from 100 for no particular reason
+    // I want to avoid processId 0 and 1 as they are
+    // "reserved" on Windows (0) and Linux (both 0 and 1)
+    private int _lastProcessId = 100;
+
+    public FakeProcess CurrentProcess { get; }
+    public List<FakeProcess> Processes { get; } = new();
+    public int LastProcessId => _lastProcessId;
```

Comment:
Replace by `get; private set;` property?

Reply:
Would be better to keep as is, but replace make _lastProcessid an Id, and call .Next() on it. Because that encapsulates the  work we do with interlocked.increment.

Reply:
I removed the last process id property entirely, and used SequentialId.